packages/common/actions/entities/team.js

```diff
@@ -264,3 +268,23 @@ export const updateTeamMembersWithRoles = (teamId, members) => ({
     members,
   },
 });
+
+export const togglePinTeamRequest = (teamId, shouldPinned) => ({
+  type: TOGGLE_PIN_TEAM_REQUEST,
+  payload: {
+    teamId,
+    shouldPinned,
+  },
+});
+
+export const togglePinTeamSuccess = (teamId, pinned) => ({
```

Comment:
Suggest to use `isPinned `instead of `pinned`

Reply:
Updated